packages/internals/src/cli/getSchema.ts

```diff
@@ -193,6 +199,10 @@ async function getSchemaWithPathInternal(
   }
 
   // 2. Try the `schema` from `PrismaConfig`
+  const prismaConfigResult = await readSchemaFromPrismaConfigBasedLocation(cwd, schemaPathFromConfig)
```

Comment:
`cwd` shouldn't be taken into account. The path provided by `defineConfig` is absolute.